src/events/janus_rabbitmqevh.c

```diff
@@ -87,6 +87,7 @@ static GThread *handler_thread;
 static GThread *in_thread;
 static void *jns_rmqevh_hdlr(void *data);
 static void *jns_rmqevh_hrtbt(void *data);
+int janus_rabbitmqevh_tryconnect(void);
 int janus_rabbitmqevh_connect(void);
```

Comment:
I'm not sure why there's two functions here now. You made it inline in the transport, why not do the same in the event handler?